grails-forge-core/src/main/java/org/grails/forge/build/dependencies/Scope.java

```diff
@@ -30,13 +30,14 @@ public class Scope {
     public static final Scope COMPILE_ONLY = new Scope(Source.MAIN, Collections.singletonList(Phase.COMPILATION));
     public static final Scope CONSOLE = new Scope(Source.MAIN, Collections.singletonList(Phase.CONSOLE));
     public static final Scope DEVELOPMENT_ONLY = new Scope(Source.MAIN, Collections.singletonList(Phase.DEVELOPMENT_ONLY));
-    public static final Scope RUNTIME = new Scope(Source.MAIN, Collections.singletonList(Phase.RUNTIME));
-    public static final Scope TEST = new Scope(Source.TEST, Arrays.asList(Phase.COMPILATION, Phase.RUNTIME));
+    public static final Scope RUNTIME_ONLY = new Scope(Source.MAIN, Collections.singletonList(Phase.RUNTIME));
+    public static final Scope TEST_IMPLEMENTATION = new Scope(Source.TEST, Arrays.asList(Phase.COMPILATION, Phase.RUNTIME));
     public static final Scope TEST_ANNOTATION_PROCESSOR = new Scope(Source.TEST, Collections.singletonList(Phase.ANNOTATION_PROCESSING));
     public static final Scope TEST_COMPILE_ONLY = new Scope(Source.TEST, Collections.singletonList(Phase.COMPILATION));
-    public static final Scope TEST_RUNTIME = new Scope(Source.TEST, Collections.singletonList(Phase.RUNTIME));
+    public static final Scope TEST_RUNTIME_ONLY = new Scope(Source.TEST, Collections.singletonList(Phase.RUNTIME));
```

Comment:
Should we also add the `integrationTest*` scopes for completeness. Geb, Selenium and perhaps some others should preferably be moved to those.

And what about renaming `compile` to `implementation`?

Reply:
Good catch, I've renamed `compile` to `implementation`.

For `integrationTest*` scopes, I'd like to do that on the `7.0.x` branch since it already has INTEGRATION_TEST_IMPLEMENTATION_TEST_FIXTURES https://github.com/grails/grails-forge/blob/a875736a7a41f108ad5a714e70acf2bd5688d3e9/grails-forge-core/src/main/java/org/grails/forge/build/dependencies/Scope.java#L40